test/e2e/instance-networking.e2e.ts

```diff
@@ -35,9 +35,13 @@ test('Instance networking tab — NIC table', async ({ page }) => {
     '/projects/mock-project/vpcs/mock-vpc'
   )
 
+  // Ensure that the 'Add network interface' button is disabled when the instance is running
+  await expect(page.getByRole('button', { name: 'Add network interface' })).toBeDisabled()
+
   // Have to stop instance to edit NICs
   await stopInstance(page)
 
+  await expect(page.getByRole('button', { name: 'Add network interface' })).toBeEnabled()
   await page.click('role=button[name="Add network interface"]')
```

Comment:
Line 45 is also the same selector, just done as a legacy string. might as pull out `const addNicButton` to use in all three spots.